x/staking/keeper/delegation.go

```diff
@@ -314,6 +328,21 @@ func (k Keeper) HasReceivingRedelegation(ctx sdk.Context,
 	return false
 }
 
+// HasMaxRedelegationEntries - redelegation has maximum number of entries
+func (k Keeper) HasMaxRedelegationEntries(ctx sdk.Context,
+	delegatorAddr sdk.AccAddress, validatorSrcAddr,
+	validatorDstAddr sdk.ValAddress) bool {
+
+	red, found := k.GetRedelegation(ctx, delegatorAddr, validatorSrcAddr, validatorDstAddr)
+	if !found {
+		return false
+	}
+	if len(red.Entries) >= int(k.MaxEntries(ctx)) {
```

Comment:
```suggestion
	if len(red.Entries) >= int(k.MaxEntries(ctx))
```

Reply:
I don't understand this?

Reply:
%s/if/return/

Reply:
Instead of checking the `bool` and then returning, why not just return the `bool`?

Reply:
thanks for fixing the typo in your suggested code comment